flexinput/build.gradle

```diff
@@ -19,10 +19,12 @@ android {
 
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
```

Comment:
this line isn't needed iirc since we don't bundle and jars or libs.

Reply:
yep your right